### DISCUSSION TITLE
Browseable API doesn't show the fancy html form for PATCH requests

### DISCUSSION THREAD

**Discussion by ionelmc** [encode/django-rest-framework#8862] [Potential Issue] [answered]
PATCH requests show only the the raw form in the browseable api (that one with just the content type and the text field). In contrast PUT requests get the form with all the fields from the serializer.

I would like to understand why there's only handling for PUT/POST, eg: https://github.com/encode/django-rest-framework/blob/22d206c1e0dbc03840c4d190f7eda537c0f2010a/rest_framework/renderers.py#L689-L696

Is this just an oversight or there's something more complicated that prevents just adding a `'patch_form': self.get_rendered_html_form(data, view, 'PATCH', request),` in there?

**Comment by baseplate-admin** [comment]
I actually also want to know the reasoning behind this decision.

Should we raise an issue to get more attention ?

<sub>Or maybe a Pull Request addressing this would be more efficient</sub>

**Comment by auvipy** [comment, marked as answer]
I don't know the exact reason behind this, but if this is nothing super complicated and don't break any existing working functionality & don't add any new public API surface, then I think we could consider a patch to improve usability in this area.

**Reply by baseplate-admin** [reply]
<sub>Thanks, Its really nice to see DRF being actually maintained. I was really frustrated to see DRF not being maintained for a long time.</sub>

<sub>also i am personally a big fan of you</sub>

So should we raise a issue from this discussion or a Pull Request?

**Reply by auvipy** [reply]
A pull request would be great & reduce redundant process

**Reply by baseplate-admin** [reply]
In which case i will wait for @ionelmc, input before taking a shot at this myself :)

**Reply by ionelmc** [reply]
Feel free to take a shot, I am swamped :-)
